engine/admin-api/domain/entity/user_activity.go

```diff
@@ -7,32 +7,24 @@ import (
 type UserActivityType string
 
 const (
-	UserActivityTypeLogin                      UserActivityType = "LOGIN"
-	UserActivityTypeLogout                     UserActivityType = "LOGOUT"
-	UserActivityTypeCreateRuntime              UserActivityType = "CREATE_RUNTIME"
-	UserActivityTypeCreateVersion              UserActivityType = "CREATE_VERSION"
-	UserActivityTypePublishVersion             UserActivityType = "PUBLISH_VERSION"
-	UserActivityTypeUnpublishVersion           UserActivityType = "UNPUBLISH_VERSION"
-	UserActivityTypeStartVersion               UserActivityType = "START_VERSION"
-	UserActivityTypeStopVersion                UserActivityType = "STOP_VERSION"
-	UserActivityTypeUpdateSetting              UserActivityType = "UPDATE_SETTING"
-	UserActivityTypeUpdateVersionConfiguration UserActivityType = "UPDATE_VERSION_CONFIGURATION"
-	UserActivityTypeCreateUser                 UserActivityType = "CREATE_USER"
-	UserActivityTypeRemoveUsers                UserActivityType = "REMOVE_USERS"
-	UserActivityTypeUpdateAccessLevels         UserActivityType = "UPDATE_ACCESS_LEVELS"
-	UserActivityTypeRevokeSessions             UserActivityType = "REVOKE_SESSIONS"
-	UserActivityTypeGenerateAPIToken           UserActivityType = "GENERATE_API_TOKEN"
-	UserActivityTypeDeleteAPIToken             UserActivityType = "DELETE_API_TOKEN"
+	UserActivityTypeCreateRuntime            UserActivityType = "CREATE_RUNTIME"
```

Comment:
This should be changed to:

```suggestion
	UserActivityTypeCreateProduct            UserActivityType = "CREATE_PRODUCT"
```